cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -86,6 +86,8 @@ This application demonstrates:
   and the other using K-Means. The application is packaged with the two possible application config JSON files at
   *resources/wikipedia-kmeans.json* and *resources/wikipedia-lda.json* in the application directory.
 
+- **The use of runtime arguments** in programs to customize for specific use cases. A namespace runtime argument can be
+  provided to access datasets from a different namespace than the one where the application is deployed.
```

Comment:
I think we should give more information than this. 
1. We should have this as cross namespace data access rather than runtime arguments. Runtime args are just a mean of providing it.
2. We should say which all programs can take namespace as runtime arguments/preferences.
3. The key which should be used to specify it.